blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/search-box.scss

```diff
@@ -0,0 +1,4 @@
+.search-box--right-absolute-positioned {
```

Comment:
Looks like this is causing issues when you have the search box configured on the left side

Existing
<img width="197" alt="Microsoft Edge-2021-09-29-11-26 36" src="https://user-images.githubusercontent.com/868127/135253720-ba6acd3e-0c06-4371-b2db-5afdb732eba0.png">

With this change -
<img width="170" alt="Microsoft Edge-2021-09-29-11-26 53" src="https://user-images.githubusercontent.com/868127/135253722-c08c1496-dbb5-4abb-8279-f4c4e7432076.png">

Reply:
thanks @matthewroach -- updated. only displaying on open now

<img width="184" alt="Screen Shot 2021-09-29 at 09 21 48" src="https://user-images.githubusercontent.com/5950956/135288078-d8d87ff3-375f-481f-9dde-1e79e2c26cf2.png">
<img width="92" alt="Screen Shot 2021-09-29 at 09 21 40" src="https://user-images.githubusercontent.com/5950956/135288085-ca065d3d-4eac-4fef-a522-19460e216130.png">